package.json

```diff
@@ -282,13 +291,15 @@
         "tabbable": "^1.1.3",
         "ts-loader": "^6.2.1",
         "typescript": "^3.7.5",
+        "uniqid": "^5.2.0",
```

Comment:
Can we reuse lodash/uniqueID? or is this a guid generator?

Reply:
I was hesitant to use lodash's, since it is numeric and counts upward per run. this `uniqid` package produces strings like 'a38gckeaa3fkqm2', but is only used for uniqueness for a temp directory. I can switch to using uuid

Reply:
its fine